client/layout/masterbar/oauth-client.scss

```diff
@@ -627,7 +621,7 @@
 		box-shadow: 0 0 0 2px var(--color-primary-10);
 	}
 
-	.button:not(.social-buttons__button) {
```

Comment:
`.button.social-buttons__button` do not exist anymore, they were all migrated to core buttons in https://github.com/Automattic/wp-calypso/pull/100807